### ISSUE TITLE
Remove empty lines around filename 🚀 

### ISSUE THREAD

**Issue by justjake** [dandavison/delta#1636]
I am trying to make `delta` more compact in its use of vertical space. I've removed the decoration for the filename, but it still prints an empty line above and below the filename. Is there any way to remove this whitespace?

![image](https://github.com/dandavison/delta/assets/296279/1129095a-bdb1-4b60-a3cc-14542d450a27)

**Comment by Gaweringo** [comment]
I am in the same boat, I like the syntax highlighting and diff highlighting from delta, but delta uses so much space with all the divider lines.
My dream would be to have the highlighting from delta with the layout/design from difftastic. So if you got any further on this, I would like to know how far you got.

**Comment by th1000s** [comment]
Yes, that should become configurable. Related, making the divider lines work with  full-width characters #1727

**Comment by tgross35** [comment]
Sort of related: I think it would be nice if `delta` had a CLI option to remove the frames, similar to what `bat` does. Filed this as https://github.com/dandavison/delta/issues/2064

**Comment by dandavison** [comment]
> I like the syntax highlighting and diff highlighting from delta, but delta uses so much space with all the divider lines.

@Gaweringo have you looked through the config options? `delta -h` is a good way to browse them. I think the various lines can all be disabled?

**Comment by Gaweringo** [comment]
I currently have these options set, which remove all the lines, but still has the extra space around the filename:
```gitconfig
[delta]
	navigate = true
	hyperlinks = true
	side-by-side = true
	line-number = true
	commit-decoration-style = none
	file-decoration-style = none
	hunk-header-decoration-style = none

	line-numbers-left-format = {nm:^4}
	line-numbers-right-format = {nm:^4}
```
